### ISSUE TITLE
site.options.dev not set on _config.js when using --dev

### ISSUE THREAD

**Issue by fserb** [lumeland/lume#86]
Is supporting `--dev` specific config a thing?

It seems that `site.options.dev` is always false inside `_config.js` and it's only set afterwards.

**Comment by oscarotero** [comment]
`dev` mode is used to render pages in `draft`. And you can set it in `_config.js` by `const site = lume({ dev: true })` or override with the `--dev` option in CLI.

**Comment by fserb** [comment]
yeah, I get that. But wouldn't it make some sense for it to be set on the `lume()` call, so it can be used inside _config?
For the a case like this:
```
const site = lume();

site.options.location = new URL(site.options.dev ? "http://localhost" : "https://mysite.com");
```

that can be controlled by the command line flag.

I'm not 100% sure if that's a reasonable use case.

**Comment by oscarotero** [comment]
For this specific use case, you can set a different location from CLI (`--location=url`). But I'm agree that dev configuration could be useful in `_config.js`.

Maybe a environment variable, like `DENO_ENV` (like `NODE_ENV`) or `LUME_DEV` (specific for lume) could help.

**Comment by valtlai** [comment]
@fserb FYI, this is fixed in [0.22.2](../blob/master/CHANGELOG.md#0222---2021-06-07) (released 2021-06-07).
